tsconfig.json

```diff
@@ -62,5 +62,7 @@
 
     /* Advanced Options */
     "forceConsistentCasingInFileNames": true  /* Disallow inconsistently-cased references to the same file. */
-  }
+  },
+  "include": ["./src/**/*"],
+  "exclude": ["./src/**/host/**/*"]
```

Comment:
Why do we exclude the host?

Reply:
the webpack server can parse tsx files. We originally only wanted to compile the cli files to JS so that it could be run from the `.bin` folder. I saw no need to build this in the build folder. Do you think we need to compile here?

Reply:
Oh ok, I understand now